CharlieBackend.Core/DTO/Homework/HomeworkRequestDto.cs

```diff
@@ -9,6 +9,7 @@ public class HomeworkRequestDto
     {
         public DateTime? DueDate { get; set; }
 
+        [Required]
```

Comment:
Does TaskText actualy must be required? Homework may be without any explanation

Reply:
@Hlib-mats yes , we need this field. it cannt be like you writed, because mentor must write task for student and TaskTest exactly for this .

Reply:
okay, persuasively